fenix/app/src/main/AndroidManifest.xml

```diff
@@ -43,7 +47,6 @@
     <application
         android:name=".FenixApplication"
         android:allowBackup="false"
-        android:extractNativeLibs="true"
```

Comment:
I see this is now useLegacyPackaging, https://developer.android.com/guide/topics/manifest/application-element#extractNativeLibs - lgtm.